components/channel_info_modal/channel_info_modal.test.tsx

```diff
@@ -6,15 +6,56 @@ import {Modal} from 'react-bootstrap';
 import {shallow} from 'enzyme';
 
 import {mountWithIntl} from 'tests/helpers/intl-test-helper';
-import ChannelInfoModal from 'components/channel_info_modal/channel_info_modal.jsx';
+import ChannelInfoModal from 'components/channel_info_modal/channel_info_modal';
+import { Channel } from 'mattermost-redux/src/types/channels';
```

Comment:
The linting checks for this PR are failing. I think it might be something to do with the spacing of these curly brackets.

Reply:
Thanks for the reminder! I think I fixed all of the lint issues but will circle back to it again if the build fails.

Reply:
Seems like I missed some lint errors again but these should be fixed now.

For the `type-check` stage, the fix for this has been merged. How should I go about upgrading the `mattermost-redux` package in `package-lock.json`? Running `npm upgrade mattermost-redux` did not change the version commit for me.

Reply:
You can run `npm install mattermost/mattermost-redux#commithash` with the merge commit from your PR into master. NPM really doesn't like working with commit hashes, so it doesn't update automatically unfortunately. We're working on getting away from that since it's kind of annoying like that.